src/blocks/card.css

```diff
@@ -0,0 +1,113 @@
+.container {
+  width: max-content;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: max-content;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.card__image {
+  margin-bottom: 40px;
+}
+
+.title {
+  width: 166px;
+  line-height: 18px;
+
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__title {
+  margin-bottom: 4px;
+}
+
+.product-code {
+  font-size: 10px;
+  color: #616070;
+  line-height: 14px;
+  width: 166px;
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+}
+
+.card__rating {
+  margin-bottom: 24px;
+}
+
+.rating__reviews {
+  font-size: 10px;
+  color: #060b35;
+}
+
+.price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+}
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.price__title {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.price__value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__buy-button {
+  width: 166px;
+  height: 40px;
+
+  font-size: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 700;
+  color: #fff;
+  background-color: #00acdc;
+
+  border: transparent;
+  border-radius: 5px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
I propose to make button inline-block element. 
Also, we can set the same height and line-height, plus add text-align: center.